cmd/minikube/cmd/start.go

```diff
@@ -304,11 +304,43 @@ func runStart(cmd *cobra.Command, args []string) {
 	}
 
 	validateSpecifiedDriver(existing)
-	ds := selectDriver(existing)
+	ds, alts, specified := selectDriver(existing)
+	err = startWithDriver(cmd, ds, existing)
+	if err != nil && !specified {
+		// Walk down the rest of the options
+		for _, alt := range alts {
+			out.WarningT("Startup with {{.old_driver}} driver failed, trying with {{.new_driver}}.", out.V{"old_driver": ds.Name, "new_driver": alt.Name})
```

Comment:
Let's start by outputting the error message. It's unlikely to be terribly long, and it should be very informative.